queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+/*Milestone 01 */
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+SELECT *
+FROM animals
+WHERE date_of_birth BETWEEN '01-01-2016' AND '12-12-2019';
```

Comment:
- [ ] You should list only the name of all animals born between 2016 and 2019, kindly replace `*` with `name` and change the second date from `12-12-2019` to `2019-31-12` so you select the whole year